packages/arb-token-bridge-ui/src/components/TransactionHistory/PendingTransactions.tsx

```diff
@@ -71,6 +73,11 @@ export const PendingTransactions = ({
     ? 'bg-gray-dark'
     : 'bg-ocl-blue'
 
+  // Show from 6th November 2023 to 7th November 2023
+  const showSubgraphMaintenanceMessage =
+    dayjs().startOf('day').isAfter(dayjs('2023-11-06').startOf('day')) &&
```

Comment:
Currently, this conditional will be browser time dependent. We need to handle the exact UTC time here since it needs to be consistent across all timezones.

Reply:
Also, we might want to show this warning a day before the actual maintenance period so that users are prepared.

Reply:
good point, updated